### ISSUE TITLE
Rendering Error in hyper-spectral images Module

### ISSUE THREAD

**Issue by michaelweinold** [ksmet1977/luxpy#10]
I am unable to reproduce the rendered image from the example given in section 4.4 of the [documentation article](https://doi.org/10.1080/15502724.2018.1518717), using the [same sample image](https://github.com/wasserwaage/CRI_Simulation/blob/master/testimage1.jpg).

The example code I ran:

```
import luxpy
from luxpy import hypspcim
import luxpy.toolboxes.spdbuild as spb 

S = spb.spd_builder(peakwl = [450,530,610],
                  fwhm = [10,10,10],
                  target = 4000,
                  tar_type = 'cct')

img_hyp = hypspcim.render_image(img = 'testimage1.jpg',
                                cspace = 'ipt',
                                spd = S,
                                D = 1,
                                show = True,
                                stack_test_ref = 21)
```

gives me [this image](https://github.com/wasserwaage/CRI_Simulation/blob/master/output.png).

**Comment by ksmet1977** [comment]
Hi Michael,Yes I noticed something similar last week. I have updated the luxpy on github (use a pip install update directly from github). I'm preparing a new version to upload to pypi in the next few days.Regards,KevinSent from my Samsung Galaxy smartphone.
-------- Original message --------From: Michael <notifications@github.com> Date: 19/12/19  12:28  (GMT+01:00) To: ksmet1977/luxpy <luxpy@noreply.github.com> Cc: Subscribed <subscribed@noreply.github.com> Subject: [ksmet1977/luxpy] Rendering Error in hyper-spectral images Module (#10) I am unable to reproduce the rendered image from the example given in section 4.4 of the documentation article, using the same sample image.
The example code I ran:
import luxpy
from luxpy import hypspcim
import luxpy.toolboxes.spdbuild as spb 

S = spb.spd_builder(peakwl = [450,530,610],
                  fwhm = [10,10,10],
                  target = 4000,
                  tar_type = 'cct')

img_hyp = hypspcim.render_image(img = 'testimage1.jpg',
                                cspace = 'ipt',
                                spd = S,
                                D = 1,
                                show = True,
                                stack_test_ref = 21)

gives me this image.

—You are receiving this because you are subscribed to this thread.Reply to this email directly, view it on GitHub, or unsubscribe.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/ksmet1977/luxpy/issues/10?email_source=notifications\u0026email_token=ACS4LXYHYSA2I6JXSBRYFC3QZNLGLA5CNFSM4J5DS4OKYY3PNVWWK3TUL52HS4DFUVEXG43VMWVGG33NNVSW45C7NFSM4IBTUQYA",
"url": "https://github.com/ksmet1977/luxpy/issues/10?email_source=notifications\u0026email_token=ACS4LXYHYSA2I6JXSBRYFC3QZNLGLA5CNFSM4J5DS4OKYY3PNVWWK3TUL52HS4DFUVEXG43VMWVGG33NNVSW45C7NFSM4IBTUQYA",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]
